doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md

```diff
@@ -123,3 +124,15 @@ F3D raytracing and exodus plugin are not working on macOS silicon yet, see this
 > I'm unable to get coloring right with step files
 
 F3D on macOS does not support coloring on cells because of a [VTK issue](https://gitlab.kitware.com/vtk/vtk/-/issues/18969).
+
+## Python
+
+> I can't find usb and vdb plugin after installing f3d with pip
+
+usd and vdb plugin are not shipped in the python wheels for now, you can compile F3D yourself with them though.
```

Comment:
```suggestion
> I can't find USD and VDB plugin after installing f3d with pip

USD and VDB plugin are not shipped in the python wheels for now, you can compile F3D yourself with them though.
```
"usb" typo as already pointed out by @Meakk + acronyms are capitalized earlier in the document.

Reply:
the plugin names are not capitalized I think ?

@Meakk

Reply:
I see, the capitalized one are the format names, my bad, ignore me then :D
Would it make sense to backtick-quote this stuff tho? (`usd` plugin, `f3d` package, `pip`, ...)

Reply:
I agree with @snoyer, let's backtrack quote this